src/plugin/scoring-metadata/src/test/org/apache/nutch/scoring/metadata/MetadataScoringFilterTest.java

```diff
@@ -0,0 +1,113 @@
+package org.apache.nutch.scoring.metadata;
```

Comment:
Nice to have a test! Alone it isn't executed running ant. Test classes must be named "Test*.java".

To run only the scoring-metadata unit tests:
```
ant test-plugin -Dplugin=scoring-metadata
```

Reply:
Thanks, Sebastian, I will address your comments.  Do you have any time frame when this should be done? If you plan to merge this into a release, then I can try to get it done by this weekend.

Reply:
Great! I'll move NUTCH-1403 to the Nutch 1.17 tasks, as I hope to get the 1.16 release candidate ready today or tomorrow. No hurry!

Reply:
@aalbahem are you able to update this patch. It would be a shame for it to sit forever. Great work, I can review it if you can address @sebastian-nagel 's comments.

Reply:
Yes, I can fix them. I happen to be too busy with my PhD thesis. I have more free time now. I can do that.

Reply:
Wow that was quick. All the best with your Ph.D. defense.

Reply:
@lewismc I have addressed the comments. Let me know if you need anything else.

Reply:
Excellent @aalbahem I'll begin reviewing. Thank you